libs/libc/unistd/Kconfig

```diff
@@ -117,3 +117,25 @@ config OPEN_MAX
 	---help---
 		The maximum number of files that a process can have open
 		at any time.  Must not be less than _POSIX_OPEN_MAX.
+
+config NAME_MAX
```

Comment:
could we add LIBC_ prefix

Reply:
emm... which should be need refresh defconfig again

Reply:
yes:(

Reply:
OK, I think I won't make the LIBC_ prefix change in this commit, maybe in the next PR, this PR just move the unistd define to libs Kconfig.